doc/Default_hotkeys.md

```diff
@@ -51,99 +61,127 @@ window list by using the first parameter.
 `Config_hotkey=#^m::Manager_minimizeWindow()`
 
+<kbd>Win</kbd><kbd>Ctrl</kbd><kbd>m</kbd>
+
 _Minimize_ the active window.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=#+m::Manager_moveWindow()`
 
+<kbd>Win</kbd><kbd>Shift</kbd><kbd>m</kbd>
+
 _Move_ the active window _by key_.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=#+s::Manager_sizeWindow()`
 
+<kbd>Win</kbd><kbd>Shift</kbd><kbd>s</kbd>
+
 _Resize_ the active window _by key_.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=#+x::Manager_maximizeWindow()`
 
+<kbd>Win</kbd><kbd>Shift</kbd><kbd>x</kbd>
+
 _Move and resize_ the active window _to_ the size of the _work area_.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=#i::Manager_getWindowInfo()`
 
+<kbd>Win</kbd><kbd>i</kbd>
+
 Get information for the active window.
 
-> The information being id, title, class, process name, style, geometry, tags 
+> The information being id, title, class, process name, style, geometry, tags
 and floating state.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=#+i::Manager_getWindowList()`
 
+<kbd>Win</kbd><kbd>Shift</kbd><kbd>i</kbd>
+
 _Get a window list_ for the active view.
 
 > The list contains information about the window id, title and class.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=!Down::View_moveWindow(0, +1)`
 
+<kbd>Alt</kbd><kbd>Down</kbd>
+
 Manually _move_ the active window _to the next area_ in the layout.
 
 > This has only an effect, if dynamic tiling is disabled (`Config_dynamicTiling=0`).
 
 -------------------------------------------------------------------------------
 `Config_hotkey=!Up::View_moveWindow(0, -1)`
 
+<kbd>Alt</kbd><kbd>Up</kbd>
+
 Manually _move_ the active window _to the previous area_ in the layout.
 
-> This has only an effect, if dynamic tiling is disabled 
+> This has only an effect, if dynamic tiling is disabled
 (`Config_dynamicTiling=0`).
 
 -------------------------------------------------------------------------------
 `Config_hotkey=!+Enter::Manager_maximizeWindow()`
 
+<kbd>Alt</kbd><kbd>Shift</kbd><kbd>Enter</kbd>
+
 _Move and resize_ the active window _to_ the size of the _work area_.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=!<n>::View_moveWindow(<n>)`
 
-Manually _move_ the active window _to the n<sup><small>th</small></sup> area_ 
+<kbd>Alt</kbd>&lt;<kbd>n</kbd>&gt;
```

Comment:
Was quite unsure how to best label this "key", so I took what I thought would be a good compromise: surrounding the n key with the < and >. Can move them inside all the same.

Reply:
The [Cheat sheet](https://github.com/fuhsjr00/bug.n/tree/master/doc/Cheat_sheet) is dependent on doc/Default_hotkeys.md. If the creation process of the cheat sheet is affected, either this or doc/Default_hotkeys.md should be adjusted.